DSNP-Wallet/Source/Modules/TestViewController.swift

```diff
@@ -131,3 +162,72 @@ extension TestViewController {
         })
     }
 }
+
+//MARK: Extrinsic Closure
+extension TestViewController {
+    func getSubscriptionIdClosure() -> ExtrinsicSubscriptionIdClosure {
+        let subscriptionIdClosure: ExtrinsicSubscriptionIdClosure = { [weak self] subscriptionId in
+            self?.viewModel?.extrinsicSubscriptionId = subscriptionId
+            
+            return self != nil
+        }
+        
+        return subscriptionIdClosure
+    }
+    
+    func getSubscriptionNotificationClosure(completion: @escaping TransactionSubscriptionCompletion) -> ExtrinsicSubscriptionStatusClosure {
+        let notificationClosure: ExtrinsicSubscriptionStatusClosure = { [weak self] result in
+            var text = ""
```

Comment:
What do you think about setting a standard, whenever we use `[weak self]`, at the top of the closure, include `guard let self else { return }` to use a non optional version of `self`

Reply:
Going to remove this logic.